siac/gatewaycmd.go

```diff
@@ -56,11 +60,13 @@ func gatewayremovecmd(addr string) {
 	fmt.Println("Removed", addr, "from peer list.")
 }
 
-func gatewaystatuscmd() {
+// gatewaylistcmd is the handler for the command `siac gateway list`.
+// Prints a list of all peers.
```

Comment:
it also prints the netaddress of the gateway...maybe that should be split into a separate command? `siac gateway` could show just the netaddress and the number of peers.